code/go/0chain.net/chaincore/chain/protocol_round_integration_tests.go

```diff
@@ -51,39 +47,6 @@ func addResultOnFinalizeRoundIfNeeded(r round.RoundI) {
 	cfg.Resulted = true
 }
 
-const (
```

Comment:
why is it removed?

Reply:
fixed on [39fc22c](https://github.com/0chain/0chain/pull/673/commits/39fc22c0e7097c8372abb8b1559e0c8c3581de47)